code/_onclick/item_attack.dm

```diff
@@ -64,11 +64,11 @@
 			else
 				return TRUE
 
-	if(force && HAS_TRAIT(user, TRAIT_PACIFISM))
+	if(HAS_TRAIT(user, TRAIT_PACIFISM) && force && user.a_intent != INTENT_HELP)
```

Comment:
Does this now let pacifists use non-lethal stun weapons like stun batons or police batons? Or do those no longer work in help intent? Either approach causes issues, imo.

Reply:
The intent (ha!) here is that pacifists can boop people with desword on help intent (and do no damage). The rest of pacifist behaviour should be unchanged.
The should still be able to use batons on help - those have `force=0` so this check fails.